pkg/reconciler/kubernetes/tektonresult/installerset.go

```diff
@@ -26,10 +26,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func (r *Reconciler) createInstallerSet(ctx context.Context, tr *v1alpha1.TektonResult) (*v1alpha1.TektonInstallerSet, error) {
+func (r *Reconciler) createInstallerSet(ctx context.Context, tr *v1alpha1.TektonResult, manifest mf.Manifest) (*v1alpha1.TektonInstallerSet, error) {
 
-	r.filterExternalDB(tr)
-	if err := r.transform(ctx, &r.manifest, tr); err != nil {
+	r.filterExternalDB(tr, manifest)
```

Comment:
it seems like here we are not receiving the value returned

Reply:
We were changing the value, so the receiver wasn't needed.

Reply:
Now I have changed the function.